### PR TITLE
hide sign in spinbox widget if only positive values

### PR DESCRIPTION
**Pull request by ScarsFun**
I need this feature in spinbox, hope will be good for everyone

### CODE REVIEW

File: src/lv_objx/lv_spinbox.c [hunk 1]
```diff
@@ -381,17 +381,20 @@ static void lv_spinbox_updatevalue(lv_obj_t * spinbox)
...
-        (*buf_p) = ' ';
+    
+    if (ext->range_min < 0) { // hide sign if there are only positive values
+       
+        /*Add the sign*/
+        (*buf_p) = ext->value >= 0 ? '+' : '-';
         buf_p++;
+
+        /*padding left*/
+        for (i = 0; i < ext->digit_padding_left; i++) {
```
Comment (kisvegabor) [inline src/lv_objx/lv_spinbox.c]: Padding should be applied independently from the sign.
So this `for` should be out of `if (ext->range_min < 0) {`

**Review by kisvegabor — COMMENTED** [review]
Just a minor thing.

File: src/lv_objx/lv_spinbox.c (hunk 1 above)
Comment (ScarsFun) [reply src/lv_objx/lv_spinbox.c]: you are right! I've corrected code.

**Comment by kisvegabor** [comment]
Great! Thank you!

**Comment by kisvegabor** [comment]
I've merged it to `dev-6.1`

**Pull request merged**